grid/chat/get-messages/profile.supr

```diff
@@ -0,0 +1,153 @@
+name = "chat/get-messages"
+version = "1.0.0"
+
+"""
+GetMessages usecase
+Retrieve paginated list of messages
+"""
+usecase GetMessages {
+	input {
+		"""
+		Destination
+		Identification of destination to retrieve messages from
+		"""
+		destination! string!
+		
+		"""
+		AfterDate
+		Date that marks beginning of messages range until current time
+		"""
+		afterDate string
+        
+		"""
+		Limit
+		Maximum number of records to retrieve
+		"""
+		limit number
+		
+		"""
+		Page
+		Identification of page to retrieve
+		"""
+		page string
+	}
+	
+	result Messages
+	error Error
+
+	example Successfull {
+		input {
+			destination = 'general',
+			limit = 2
+		}
+
+		result {
+			messages = [
+				{
+					id = '1'
+					author = 'U01',
+					text = 'success!'
+				}
+			]
+		}
+	}
+
+	example Failed {
+		input {
+			destination = 'not-existing',
+		}
+
+		error {
+			title = 'channel_not_found'
+		}
+	}
+}
+
+model Message {
+	"""
+	Id
+	Identification of message
+	"""
+	id! string!
+
+	"""
+	Author
+	Author of retrieved message
+	"""
+	author! string!
+
+	"""
+	Destination
+	Destination of retrieved message
+	"""
+	destination string!
+
+	"""
+	ThreadId
+	Thread destination of retrieved message
+	"""
+	threadId string
+
+	"""
+	Text
+	Content of retrieved message
+	"""
+	text string
+
+	"""
+	CreatedAt
+	Date retrieved message was created
+	"""
+	createdAt string
+
+	"""
+	CreatedAt
+	Date retrieved message was last updated
+	"""
+	updatedAt string
+
+	"""
+	Reactions
+	List of reactions on retrieved message
+	"""
+	reactions [
+		{
+			emoji string
```

Comment:
I am uncertain about naming the property as "emoji", but since all platforms seem to name custom ones as "custom emojis", let's keep it. However, it should be specified what I get here; is it a text alias of the emoji like `:+1:` or `+1` or a unicode identifier?

Preferably mention this info the explanation of the `reactions` property (since I think we don't render nested properties well on the web).

Reply:
Slack returns identifier of emoji between colons, so in case of 👍🏻 it returns `+1`